### ISSUE TITLE
Remaing issues with upgrade accessibility warnings to errors

### ISSUE THREAD

**Issue by dqnykamp** [Doenet/DoenetML#893]
PR #891 adds the ability to upgrade accessibility warnings to errors. However, these issues were left unresolved:

- Since upgrading accessibility warnings is currently done via a preprocessor that does not have access to ref resolutions, it does not pick up all accessibility warnings
- Currently, the warnings that are upgraded to errors are not removed. Consider removing them.
- Label or description elements, such as `<label><m>   </m></label>`, that have children with only blank children, do not trigger upgrading the warning to an error.
